### PR TITLE
En 4379-Cross-build

### PR DESCRIPTION
**Pull request by traian-schiau**

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ VERSION:
 		Value: logger.LogInfo,
 	}
 	// bootstrapRoundIndex defines a flag that specifies the round index from which node should bootstrap from storage
-	bootstrapRoundIndex = cli.UintFlag{
+	bootstrapRoundIndex = cli.Uint64Flag{
```
Comment (bogdan-rosianu) [inline cmd/node/main.go]: you can also change in main.go line 696 -> change `GlobalUint` to `GlobalUint64` and remove the cast

File: cross_build.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    echo "Building $output_name"
+    if [ $GOOS = "windows" ]; then
+        output_name+='.exe'
+    fi
+
+    pushd cmd/node
+    env GOOS=$GOOS GOARCH=$GOARCH go build -o $output_name -a -i -v -ldflags="-X main.appVersion=$APP_VER"
+    popd
+    if [ $? -ne 0 ]; then
+        echo 'An error has occurred! '
```
Comment (bogdan-rosianu) [inline cross_build.sh]: you can also echo the `$?` so we can see the error code

File: cmd/node/main.go (hunk 1 above)
Comment (traian-schiau) [reply cmd/node/main.go]: ok

File: cross_build.sh (hunk 2 above)
Comment (traian-schiau) [reply cross_build.sh]: Will not help, is just the numeric error code 
+ the compile output is on stdout already

File: cross_build.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    GOOS=${platform_split[0]}
+    GOARCH=${platform_split[1]}
+    output_name=$BASEDIR/$GOOS/$GOARCH/$package_name
+    echo "Building $output_name"
+    if [ $GOOS = "windows" ]; then
+        output_name+='.exe'
+    fi
+
+    pushd cmd/node
+    env GOOS=$GOOS GOARCH=$GOARCH go build -o $output_name -a -i -v -ldflags="-X main.appVersion=$APP_VER"
```
Comment (iulianpascalau) [inline cross_build.sh]: It seems this renders some "permission denied" errors as it tries to write in /usr/local/go/pkg dir. $GOPATH is somewhere in home.

Comment (traian-schiau) [reply cross_build.sh]: That's a go install/setup issue, in my opinion

**Pull request merged**